README.md

```diff
@@ -790,77 +792,138 @@ FOUNDRY_RESET_CONNECTIONS=connection1,connection2
-$post->setTitle('New Title');   // won't be auto-refreshed
-$post->setBody('New Body');     // won't be auto-refreshed
-$post->save();                  // save changes (auto-refreshing re-enabled)
-
-// alternatively, use "withAutoRefresh()" - auto-refreshing disabled before running callback and re-enabled after
-$post->withoutAutoRefresh(function(Proxy $post) {
-    /* @var Post $post */
-    $post->setTitle('New Title');
-    $post->setBody('New Body');
-    $post->save();
-});
+// refresh from the database
+$post->refresh();
 
-// if the first argument is type-hinted as the wrapped object, it will be passed to the closure (and not the proxy)
-$post->withoutAutoRefresh(function(Post $post) {
-    $post->setTitle('New Title');
-    $post->setBody('New Body');
-})->save();
+// delete from the database
+$post->remove();
 
+$post->repository(); // repository proxy wrapping PostRepository (see Repository Proxy section below)
+```
+
+#### Force Setting
+
+Object proxies have helper methods to access non-public properties of the object the wrap:
+
+```php
 // set private/protected properties
 $post->forceSet('createdAt', new \DateTime()); 
 $post->forceSet('created_at', new \DateTime()); // can use snake case
 $post->forceSet('created-at', new \DateTime()); // can use kebab case
 
-/**
- * CAVEAT - When force setting multiple properties, the previous set's changes will be lost because
- * of auto-refreshing. Use "disableAutoRefresh()"/"withoutAutoRefresh()" (shown above) or "forceSetAll()"
- * to overcome this.
- */
+// get private/protected properties
+$post->forceGet('createdAt');
+$post->forceGet('created_at'); // can use snake case
+$post->forceGet('created-at'); // can use kebab case
+```
+
+#### Auto-Refresh
+
+Object proxies have the option to enable *auto refreshing* that removes the need to call `->refresh()` before calling
+methods on the underlying object. When auto-refresh is enabled, most calls to proxy objects first refresh the wrapped
+object from the database.
+
+```php
+use App\Factory\PostFactory;
+
+$post = PostFactory::new(['title' => 'Original Title'])
+    ->create()
+    ->enableAutoRefresh()
+;
+
+// ... logic that changes the $post title to "New Title" (like your functional test)
+
+$post->getTitle(); // "New Title" (equivalent to $post->refresh()->getTitle())
+```
+
+Without auto-refreshing enabled, the above call to `$post->getTitle()` would return "Original Title".
+
+**NOTE**: A situation you need to be aware of when using auto-refresh is all methods refresh the object first. If
+changing the object's state via multiple methods (or multiple force-sets), the previous changes will be lost:
+
+```php
+use App\Factory\PostFactory;
+
+$post = PostFactory::new(['title' => 'Original Title', 'body' => 'Original Body'])
+    ->create()
+    ->enableAutoRefresh()
+;
+
+$post->setTitle('New Title'); // or using ->forceSet('title', 'New Title')
+$post->setBody('New Body'); // or using ->forceSet('body', 'New Body')
```

Comment:
Instead of mentioning `forceSet` here (which you already mention above), I would remove the comment from the first line. And on the second line, say:

> // this causes the object to be refresh, which means the "New Title" title was replaced by the database contents